core/api/serializers.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework.serializers import ModelSerializer
```

Comment:
Is there a reason for this file to be in a separate module? I may be wrong, but I think that we should keep to the standard file structure.